{
  if (isFailing) {
    rollbackFailed=true;
    throw new CacheException("Induced failure");
  }
 else {
    return invokeNextInterceptor(ctx,command);
  }
}
